tests/containers/push_container_image_to_gcr.pm

```diff
@@ -40,7 +40,6 @@ sub post_fail_hook {
 
 sub post_run_hook {
     my ($self) = @_;
-    $self->{provider}->delete_image();
```

Comment:
Why?

Reply:
We need to mantain this image on the repository to be used by the kubernetes test.